decidim-core/app/models/decidim/user.rb

```diff
@@ -47,6 +47,7 @@ def self.all
     validates :tos_agreement, acceptance: true, allow_nil: false, on: :create
     validates :tos_agreement, acceptance: true, if: :user_invited?
     validates :email, :nickname, uniqueness: { scope: :organization }, unless: -> { deleted? || managed? || nickname.blank? }
+    validates :time_zone, presence: true, time_zone: true
```

Comment:
```suggestion
    validates :time_zone, time_zone: true, if:-> { timezone.present? }
```
We don't need to enforce timezones here